libsolidity/formal/SMTEncoder.cpp

```diff
@@ -304,10 +307,49 @@ void SMTEncoder::endVisit(Block const& _block)
 
 bool SMTEncoder::visit(InlineAssembly const& _inlineAsm)
 {
+	struct AssignedExternalsCollector: public yul::ASTWalker
+	{
+		using ExternalReferences = map<yul::Identifier const*, InlineAssemblyAnnotation::ExternalIdentifierInfo>;
```

Comment:
This seems to be only used once, so do we really need the using?

Reply:
Removed the using.